eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -151,12 +151,17 @@ stages:
           Codeql.Enabled: true
           Codeql.BuildIdentifier: ${{ parameters.ServiceDirectory }}
           Codeql.SkipTaskAutoInjection: false
+          skipComponentGovernanceDetection: true
```

Comment:
Why not import globals.yml here?

Reply:
It looks like globals is specifically not imported because of the overridden codeql variables here. IIRC if we import globals first the overrides will still take effect?

Reply:
Correct the last one wins.

Reply:
Ok I'll switch then